### PR TITLE
{AppService}: Updating websites to use the latest SDK version

### PR DESCRIPTION
**Pull request by panchagnula**
**Description**  
This PR updates Az CLI to use the latest version of Python SDK for websites.
This PR doesn't change the api-version or any breaking changes but has support for some new features that we want to build.

**Testing Guide**  
Ran the existing test cases using adzes test appservice

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

**Comment by panchagnula** [comment]
FYI @ElleTojaroon

**Comment by yungezz** [comment]
hi @qwordy could you pls help to review? thanks

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -10,6 +10,10 @@ Release History
 
 * `az acr check-health`: Fix "DOCKER_PULL_ERROR" on Windows (#13158)
 
+**AppService**
+
+* `az webapp up`: Fixing case for creating RG is failing when location is not provided
```
Comment (qwordy) [inline src/azure-cli/HISTORY.rst:15]: Do not modify HISTORY.rst manually. Put it in PR title or PR description. HISTORY.rst will be generated when a new version is going to release.
https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests

Comment (panchagnula) [reply src/azure-cli/HISTORY.rst:15]: Got it.

File: src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py [hunk 2]
```diff
@@ -182,11 +179,13 @@ def test_webapp_up_dotnetcore_e2e(self, resource_group):
             os.sep + os.sep, os.sep), up_working_dir)
         self.assertTrue(result['runtime_version'] == 'dotnetcore|2.2')
         self.assertTrue(result['os'].lower() == 'windows')
+        self.assertNotEqual(result['location'], 'None')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py:182]: There is a method `assertIsNotNone`

**Pull request merged**